src/matlab2tikz.m

```diff
@@ -1278,11 +1278,14 @@ function addCustomCode(fid, before, code, after)
 
     keywordTickLabelMode = [upper(axis), 'TickLabelMode'];
     tickLabelMode = get(handle, keywordTickLabelMode);
-    keywordTickLabel = [upper(axis), 'TickLabel'];
-    tickLabels = cellstr(get(handle, keywordTickLabel));
     if strcmpi(tickLabelMode, 'auto') && ~m2t.cmdOpts.Results.strict
         pgfTickLabels = [];
     else % strcmpi(tickLabelMode,'manual') || m2t.cmdOpts.Results.strict
+        keywordTickLabel = [upper(axis), 'TickLabel'];
+        interpreter = 'none'; % MATLAB only supports 'none'
```

Comment:
In HG1, that seems to be the case.
However, in HG2 the [interpreter for axes](http://www.mathworks.com/matlabcentral/answers/102053#comment_300079) can be set (and the default is seems to be the `tex` interpreter).

I think we can just replace the line
`interpreter = 'none';` 
with
```matlab
interpreter = getOrDefault(handle, 'TickLabelInterpreter', 'none');
% HG2 allows to set this property, HG1 tacitly uses the interpreter 'none'
% See http://www.mathworks.com/matlabcentral/answers/102053#comment_300079
```